### PR TITLE
Add lexer for PsySH console for PHP

### PR DESCRIPTION
**Pull request by ramsey**
This lexer is based on the PythonConsoleLexer and provides the ability
to highlight console input and output for PsySH, a developer console and
REPL for PHP. See https://psysh.org.

### CODE REVIEW

File: pygments/lexers/php.py [hunk 1]
```diff
@@ -85,6 +87,56 @@ class ZephirLexer(RegexLexer):
...
+    .. sourcecode:: psysh
+
+        >>> $greeting = function($name): string {
+        ...     return "Hello, {$name}";
+        ... };
+        => Closure($name): string {#2371 …3}
+        >>> $greeting('World')
+        => "Hello, World"
+
+    .. _PsySH: https://psysh.org/
```
Comment (Anteru) [inline pygments/lexers/php.py:103]: Please add a `.. versionadded:: 2.7` note as well.

Comment (ramsey) [reply pygments/lexers/php.py:103]: I've made the requested addition. Sorry it took so long to get to. 😁

**Comment by ramsey** [comment]
Following up to see if there are any other changes you'd like. Thanks!

**Comment by Anteru** [comment]
Nope, all good -- thanks for the contribution!

**Comment by ramsey** [comment]
Thanks!

**Pull request merged**